reference/uitpas.json

```diff
@@ -3164,26 +3253,66 @@
           "Models"
         ],
         "description": "The `Pass` entity includes basic information about the UiTPAS and its related `Passholder`.  ",
+        "x-examples": {
+          "Example": {
+            "passholderId": "b34c19fc-bcf9-4d10-aeb2-6bc9efd38b68",
+            "uitpasNumber": "0560002524314",
+            "points": 12,
+            "postalCode": "1000",
+            "messages": [
+              {
+                "level": "WARNING",
+                "text": "Je sociaal tarief is verlopen. Contacteer je UiTPAS balie."
+              }
+            ]
+          }
+        },
         "properties": {
           "passholderId": {
             "type": "string",
+            "description": "ID of the passholder associated with this pass",
             "readOnly": true
           },
           "uitpasNumber": {
             "type": "string",
+            "description": "UiTPAS number of this pass",
             "readOnly": true
           },
           "points": {
             "type": "integer",
+            "description": "Number of points of the passholder of this pass",
             "readOnly": true
           },
           "postalCode": {
             "type": "string",
+            "description": "Postal code of the passholder of this pass",
             "readOnly": true
           },
-          "socialTariffExpired": {
-            "type": "boolean",
-            "description": "If the social tariff of the passholder is expired, this field will be `true`. ",
+          "messages": {
+            "type": "array",
+            "description": "Message that, if present, must be displayed to the user",
+            "items": {
+              "type": "object",
+              "properties": {
+                "level": {
+                  "type": "string",
+                  "enum": [
+                    "WARNING",
+                    "INFO"
```

Comment:
It's probably best to use some kind of standardized severity levels here.
For example we could use:

```
INFO
WARN
ERROR
```

(I've added `ERROR` here in case we ever need it, so integrators already know it)

From: https://sematext.com/blog/logging-levels/

This seems to be a de-facto standard of log levels.